tests/storage/README.md

```diff
@@ -45,8 +49,7 @@ environment are skipped automatically. The tests must deal with the
 missing storage configuration by detecting that the storage does not
 exist.
 
-To add new storage configurations edit the STORAGE list in the
-userstorage.py tool.
+To add new storage configurations edit the BACKENDS list in storage.py.
 
 
 ## How user storage works?
```

Comment:
Maybe delete this and point to the upstream documentation?

If this is not documented upstream, maybe move this documentation
upstream.

Reply:
True, it is documented upstream. I'll link it.